src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="DeployExtension">
```

Comment:
So that the explicit `DeployExtension=false` required for this project (since it doesn't include a source extension manifest file) takes precedence over any user defined option.